pelican/tests/test_settings.py

```diff
@@ -319,3 +320,11 @@ def test_coerce_overrides(self):
             'THEME_STATIC_DIR': 'theme',
         }
         self.assertDictEqual(overrides, expected)
+
+    def test_stringify_plugins(self):
+        settings = {
+            'PLUGINS': ['string_plugin', NoopPlugin]
```

Comment:
Well, technically, due to the way you structured `NoopPlugin` this won't "work". You need an instance (`NoopPlugin()`).